backpack.tf extended sorting.user.js

```diff
@@ -1268,4 +1268,246 @@ class</a></li>
 		childList: true
 	});
 
+    // ################# Killstreak Kit Additions for Stats pages ##############
+
+    function trimSpaces(string){
+        return string.replace(/^\s+/g, '').replace(/\s+$/g, '').replace(/\s+/g,"%20");
+    }
+
+    function appendKSToPath(tier,path,currentTier){
+        const ksTierDict = {
+            1 : "Killstreak%20",
+            2 : "Specialized%20Killstreak%20",
+            3 : "Professional%20Killstreak%20",
+        }
+        let splitPath = path.split("/");
+        splitPath[3] = splitPath[3].replace(ksTierDict[currentTier],"");
+        if(tier > 0){
+            splitPath[3] = ksTierDict[tier].concat("",splitPath[3]);
+        }
+        let tieredPath = splitPath.join("/");
+        return tieredPath;
+    }
+
+    class itemData{
+        constructor(headerItem){
+            this.name = headerItem.dataset.base_name;
+            this.defindex = headerItem.dataset.defindex;
+            this.killstreak_tier = headerItem.dataset.ks_tier ? headerItem.dataset.ks_tier : 0;
+            this.aussie = headerItem.dataset.australium ? headerItem.dataset.australium : 0;
+            this.slot = headerItem.dataset.slot ? headerItem.dataset.slot : "none";
```

Comment:
A shorter way to write this expression would be the following
```suggestion
            this.killstreak_tier = headerItem.dataset.ks_tier || 0;
            this.aussie = headerItem.dataset.australium || 0;
            this.slot = headerItem.dataset.slot || "none";
```